mmv1/products/certificatemanagerpublicca/ExternalAccountKey.yaml

```diff
@@ -0,0 +1,70 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'ExternalAccountKey'
+base_url: 'projects/{{project}}/locations/global/externalAccountKeys'
+create_url: 'projects/{{project}}/locations/global/externalAccountKeys'
+self_link: 'projects/{{project}}/locations/global/externalAccountKeys/{{key_id}}'
+id_format: '{{name}}'
+create_verb: :POST
+immutable: true
+skip_delete: true
+exclude_import: true
+exclude_validator: true
+description: |
```

Comment:
Let's add a warning to the page that the resource is creation-only and won't be read back from the API or deleted. Those aren't covered by our docs, I think, but here's a reference where another resource added one: https://github.com/GoogleCloudPlatform/magic-modules/blob/31993a907b4d48108c0f031f747f1c51ab098f61/mmv1/products/deploymentmanager/Deployment.yaml#L53-L66